src/tests/test_hiv_status.py

```diff
@@ -22,16 +22,18 @@ def test_initial_hiv():
 
 
 def test_hiv_update():
+    pd.set_option('display.max_columns', None)
     pop_size = 100000
-    pop = Population(size=pop_size, start_date=date(1989, 1, 1)).data
-    HIV_module = HIVStatusModule()
-    pop["HIV_status"] = HIV_module.initial_HIV_status(pop)
-    # to keep this independent just creating partners for everyone
+    pop = Population(size=pop_size, start_date=date(1989, 1, 1))
+    data = pop.data
+    prev_status = data["HIV_status"].copy()
+
     for i in range(10):
-        prev_status = pop["HIV_status"].copy()
-        pop["num_partners"] = pd.Series([2]*pop_size)
```

Comment:
Just double checking that this was left out on purpose (so the test is still meaningful without setting partners for everyone)

Reply:
Yes, I actually found that it was better to use the ordinary initialisation that the population object will do for now as it prevent people from being assigned partners if they are under 15 / not born yet. So `num_partners` is still set here, but this is no longer independent of the function that generates the number of partners. We could recreate this independence by using a better assignment in the test that takes into account things like the ages but for now checking that it worked with the real thing was useful.